src/main/java/com/muntian/passportinfile/services/TextFilePassportService.java

```diff
@@ -11,16 +11,25 @@
 
 //This class is my code
 public class TextFilePassportService implements PassportService {
+    private PassportAccessor passportAccessor;
+    private VisaAccessor visaAccessor;
+    private Collection<Passport> passports;
+    private Collection<Visa> visas;
+    private Map<String, Visa> visaMap;
+
+    public TextFilePassportService(String passportFileName, String visaFileName) {
+        this.passportAccessor = new TextFilePassportAccessor(passportFileName);
+
+        this.visaAccessor = new TextFileVisaAccessor(visaFileName);
```

Comment:
Accessor objects should be passed as arguments. It is not good to create them in constructor.